src/bp/core/config/config-loader.ts

```diff
@@ -50,6 +62,47 @@ export class GhostConfigProvider implements ConfigProvider {
     await this.ghostService.forBot(botId).upsertFile('/', 'bot.config.json', JSON.stringify(config, undefined, 2))
   }
 
+  public async createDefaultConfigIfMissing() {
+    const botpressConfig = path.resolve(process.PROJECT_LOCATION, 'data', 'global', 'botpress.config.json')
+
+    if (!fse.existsSync(botpressConfig)) {
+      await this.ensureDataFolderStructure()
+
+      const botpressConfigSchema = path.resolve(process.PROJECT_LOCATION, 'data', 'botpress.config.schema.json')
+      const defaultConfig = defaultJsonBuilder(JSON.parse(fse.readFileSync(botpressConfigSchema, 'utf-8')))
+
+      const config = {
+        $schema: `../../botpress.config.schema.json`,
+        ...defaultConfig,
+        modules: await this.getModulesListConfig()
+      }
+
+      await fse.writeFileSync(botpressConfig, JSON.stringify(config, undefined, 2))
+    }
+  }
+
+  private async ensureDataFolderStructure() {
+    const requiredFolders = ['bots', 'global', 'storage']
+    const schemasToCopy = ['botpress.config.schema.json', 'bot.config.schema.json']
+    const dataFolder = path.resolve(process.PROJECT_LOCATION, 'data')
+
+    for (const folder of requiredFolders) {
+      await fse.ensureDir(path.resolve(dataFolder, folder))
+    }
+
+    for (const schema of schemasToCopy) {
+      const schemaContent = fs.readFileSync(path.join(__dirname, 'schemas', schema))
```

Comment:
Can you add a comment here that readFileSync + writeFileSync are required because reading those files from `pkg` ?